### ISSUE TITLE
`api_url` should be easily configurable from the user's code

### ISSUE THREAD

**Issue by ebuchlin** [Helioviewer-Project/python-api#92]
`api_url` is currently hard-coded as `https://api.helioviewer.org/v2/` in `hvpy.config.Settings`, although the use of `pydantic.Field` allows this to be overriden by environment variable `HELIOVIEWER_API_URL`.

Other servers, such as the [MEDOC/IAS](https://helioviewer.ias.u-psud.fr/) server, can be available when the GSFC server is not available or not up-to-date with the latest data, or can have additional datasets (e.g. Solar Orbiter at ESA). They should also be usable with `hvpy`.

For the user, it is more convenient to have the possibility to choose the server on the fly, rather than setting an environment variable (which is [undocumented](https://hvpy.readthedocs.io/en/latest/index.html)). This was possible with `sunpy.net.helioviewer` as the server was a parameter of `HelioviewerClient()`, but it doesn't appear that it is currently possible with `hvpy`.

See also @wafels 's [comment](https://github.com/Helioviewer-Project/python-api/issues/52#issuecomment-1205414673) on #52.

**Comment by nabobalis** [comment]
Can you see if `hvpy.set_api_url` works for you.

**Comment by ebuchlin** [comment]
It works, thanks! I should have looked until the end of `config.py`, sorry. I have now seen the [reference documentation for `set_api_url()`](https://hvpy.readthedocs.io/en/latest/api/hvpy.utils.save_file.html), I wonder if there could be something about it in the [User guide](https://hvpy.readthedocs.io/en/latest/guide.html).

**Comment by nabobalis** [comment]
Yeah there are a lot of improvements to be made for the user guide, this is one of them.
